docs/source/guides/user/chapters/operations.rst

```diff
@@ -284,12 +284,13 @@ You have two ways of discovering the tests. You can simulate the execution by
 using the ``--dry-run`` argument::
 
     avocado run /bin/true --dry-run
+    (avocado) [local@laptop avocado]$ avocado run /bin/true --dry-run
```

Comment:
There are two calls of `avocado run`

Reply:
nice catch, thanks. Fixing as a force-push here.